### ISSUE TITLE
New rule: `UnnecesaryBackticks`

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#4739]
## Expected Behavior of the rule
Flag things like this:

```kotlin
fun `thisFunctionDoesntNeedBacktickts`()

class `HelloWorld`

val `fixMePleaseIdontLikeThoseBackticks`: Int
```

Because there is no need for those backticks (`` ` ``)

## Context
After we move from spek to JUnit we have a lot of unnecessary backticks in function and class names.
